lib/index.d.ts

```diff
@@ -134,13 +133,13 @@ declare namespace planck {
         m_next: Fixture | null;
         m_proxies: FixtureProxy[];
         m_proxyCount: number;
-        m_userData: unknown;
+        m_userData: any;
```

Comment:
If you use `any` you can remove the `// TypeScript Version: 3.0` line at the top.

Reply:
I undid it, sorry I'm new to TS! :)

Reply:
`any` is also correct 😄 `unknown` forces the user to either cast the returned value (if he knows the type) or check the existence of properties. I haven't used this type much as it is new to [Typescript 3.0](https://blogs.msdn.microsoft.com/typescript/2018/07/30/announcing-typescript-3-0/#the-unknown-type), so it maybe is more cumbersome compared to `any` but on the other hand enforces stricter code. Not an easy decision.